cpu/cpu_test.go

```diff
@@ -160,7 +158,7 @@ func (cis *CPUInfoSuite) TestGetMetricTypes() {
 
 				namespaces := []string{}
 				for _, m := range mts {
-					namespaces = append(namespaces, strings.Join(m.Namespace().Strings(), "/"))
+					namespaces = append(namespaces, strings.Join(m.Namespace.Strings(), "/"))
```

Comment:
@katarzyna-z That fixed the issue. Changed strings.Join(m.Namespace.Strings(), "/") to m.Namespace.String()